### DISCUSSION TITLE
How to add to mod

### DISCUSSION THREAD

**Discussion by Dog517** [Siphalor/nbt-crafting#102] [Q&A] [answered]
Hello, so Im making a mod for fabric 1.18.2, and I want to make it so that this mod is in my mod. maybe something like
 implementation 'com.github.Siphalor:nbt-crafting:2.0.0-alpha.16'
Thanks!

**Comment by Siphalor** [comment, marked as answer]
See here: https://mcwiki.siphalor.de/nbt-crafting/v2/#modders/

;)
